python/src/wslink/websocket.py

```diff
@@ -285,7 +290,7 @@ def sendWrappedMessage(self, rpcid, content, method=''):
         wrapper = {
             "wslink": "1.0",
             "id": rpcid,
-            "result": content,
+            "result": content if content else [],
```

Comment:
OK, I think the error here is in session.js: onMessage
if (payload.result) will fail for a 'falsy' result, and it won't resolve the promise. I'll fix that.